lib/caconfig.go

```diff
@@ -49,6 +49,8 @@ ca:
   certfile: ca-cert.pem
   # The CA key file
   keyfile: ca-key.pem
+  # Ignore Certificate Expiration in the case of re-enroll
+  ignoreCertExpiry: false
```

Comment:
The precedent is all lowercase.

Reply:
I'm new at fabric-ca config... when do config options go into lib/caconfig.go versus in cmd/fabric-ca-server/config.go? I looked at some of the prior config option commits, and most of the config options go into cmd/fabric-ca-server/config.go, while just the highest level config options go in lib/caconfig.go.

Reply:
What the difference is between the 2, I'm not really sure; there are no comments in the code;  the `cmd/fabric-ca-server/config.go` seems to have the more complete file, and the call stack of the other config seems to be around multicast?  Can move it.

Reply:
In terms of naming... I've seen `clientAuthRequired`  admit that `ignorereenrollcertexpiry` for me is harder to read than `ignore ReenrollCertExpiry`